### PR TITLE
Fix Healthcheck API doc

### PR DESCRIPTION
**Pull request by mkumatag**
Fixes : #32089

### CODE REVIEW

**Comment by vdemeester** [comment]
SGTM but ping @talex5 to be sure 👼

File: api/types/container/config.go [hunk 1]
```diff
@@ -12,10 +12,10 @@ type HealthConfig struct {
...
 	// An empty slice means to inherit the default.
 	// The options are:
-	// {} : inherit healthcheck
-	// {"NONE"} : disable healthcheck
-	// {"CMD", args...} : exec arguments directly
-	// {"CMD-SHELL", command} : run command with system's default shell
+	// [] : inherit healthcheck
+	// ["NONE"] : disable healthcheck
+	// ["CMD", args...] : exec arguments directly
+	// ["CMD-SHELL", command] : run command with system's default shell
```
Comment (thaJeztah) [inline api/types/container/config.go]: This is GoDoc, not JSON, so these probably should not be changed

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: api/types/container/config.go (hunk 1 above)
Comment (mkumatag) [inline api/types/container/config.go]: @thaJeztah roll-backed the changes on this file, PTAL at latest commit.

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Comment by thaJeztah** [comment]
failing CI is not related

**Comment by mkumatag** [comment]
Thanks @thaJeztah

**Pull request merged**